nrfs/include/internal/services/nrfs_reset.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef NRFS_INTERNAL_RESET_H
+#define NRFS_INTERNAL_RESET_H
+
+#include <internal/services/nrfs_generic.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief Reset request structure. */
+typedef struct __NRFS_PACKED {
+	nrfs_hdr_t hdr; /**< Header of the message. */
+	nrfs_ctx_t ctx; /**< Context of the message. */
+} nrfs_reset_req_t;
+
+/** @brief Reset service notification structure. */
+typedef struct __NRFS_PACKED {
+	nrfs_hdr_t hdr; /**< Header of the message. */
+	nrfs_ctx_t ctx; /**< Context of the message. This is left for compatibility purposes. Should always be zero. */
+} nrfs_reset_rsp_t;
```

Comment:
Looks like this request/response pattern used in many places, why not use a common structure for it?

Reply:
It's a good point, I propose to change it in the next step (after nrfs publishing).